user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.md

```diff
@@ -401,7 +401,7 @@ Configures the DataMiner System to use SNMP++ for the polling of all three SNMP
 > [!IMPORTANT]
 >
 > - This feature does not support polling of IPv6 addresses. In case IPv6 addresses are polled on your DMA, do not activate this feature.
-> - [NT_SNMP_RAW_GET](xref:NT_SNMP_RAW_GET) and [NT_SNMP_RAW_SET](xref:NT_SNMP_RAW_SET) calls do not take this soft-launch option into account yet. As a result, SNMPv1 and SNMPv2 `RawGet` and `RawSet` calls are always executed by WinSNMP.
+> - Prior to DataMiner version **TBD** (RN40019), [NT_SNMP_RAW_GET](xref:NT_SNMP_RAW_GET) and [NT_SNMP_RAW_SET](xref:NT_SNMP_RAW_SET) calls do not take this soft-launch option into account yet. As a result, SNMPv1 and SNMPv2 `RawGet` and `RawSet` calls are always executed by WinSNMP.
```

Comment:
This change has already been added with the following commit: https://github.com/SkylineCommunications/dataminer-docs/commit/a504cc2816b8f909db0f64710c3788a7f2b31053
OK to close this pull request?